content/docs/testing-environments.md

```diff
@@ -0,0 +1,57 @@
+---
+id: testing-environments
+title: Testing environments
+permalink: docs/testing-environments.html
+---
+
+<!-- This document is intended for folks who are comfortable with Javascript, and have probably written tests with it. It acts as a reference for the differences in testing environments for React components, and how those differences affect the tests that they write. This document also assumes a slant towards web-based react-dom components, but has notes for other renderers. -->
+
+Testing environments can affect the way you write and run tests for React Components. 
```

Comment:
This sounds a bit like a vague truism. "Can affect", so..?

Maybe could be a bit more concrete about what people will get out of this document.

Reply:
Doesn't the next line cover that?